test/lib/tag-parsers/res/fanatics_product.html

```diff
@@ -0,0 +1,123 @@
ref="/content/content/www.fanatics.com/apple-icon-114x114.png" sizes="114x114"/><link rel="apple-touch-icon" href="/content/content/www.fanatics.com/apple-icon-120x120.png" sizes="120x120"/><link rel="apple-touch-icon" href="/content/content/www.fanatics.com/apple-icon-144x144.png" sizes="144x144"/><link rel="apple-touch-icon" href="/content/content/www.fanatics.com/apple-icon-152x152.png" sizes="152x152"/><link rel="apple-touch-icon" href="/content/content/www.fanatics.com/apple-icon-180x180.png" sizes="180x180"/><title>LeBron James Los Angeles Lakers Jordan Brand 2021/22 #6 Swingman Player Jersey Purple - Statement Edition</title><meta charSet="UTF-8"/><meta name="viewport" content="width=device-width, initial-scale=1"/><meta content="LeBron James Los Angeles Lakers Jordan Brand 2021/22 #6 Swingman Player Jersey Purple - Statement Edition | Fanatics is #ALLIN" name="description"/><meta content="LeBron James Los Angeles Lakers Jordan Brand 2021/22 #6 Swingman Player Jersey Purple - Statement Edition" name="keywords"/><meta/><meta property="og:image" content="https://fanatics.frgimages.com/FFImage/thumb.aspx?i=/productimages/_4291000/ff_4291740-ffd05487a208f9d387d4_full.jpg"/><meta property="twitter:image" content="https://fanatics.frgimages.com/FFImage/thumb.aspx?i=/productimages/_4291000/ff_4291740-ffd05487a208f9d387d4_full.jpg"/><meta property="og:title" content="LeBron James Los Angeles Lakers Jordan Brand 2021/22 #6 Swingman Player Jersey Purple - Statement Edition"/><meta property="og:description" content="This Jordan Brand LeBron James Swingman Player Jersey will make any loyal Los Angeles Lakers fan feel like a part of the team. It features crisp Los Angeles Lakers graphics and Dri-FIT technology to provide a bold yet comfortable look. This LeBron James gear will get you pumped up and ready for game day."/><meta property="twitter:title" content="LeBron James Los Angeles Lakers Jordan Brand 2021/22 #6 Swingman Player Jersey Purple - Statement Edition"/><meta property="twitter:description" content="This Jordan Brand LeBron James Swingman Player Jersey will make any loyal Los Angeles Lakers fan feel like a part of the team. It features crisp Los Angeles Lakers graphics and Dri-FIT technology to provide a bold yet comfortable look. This LeBron James gear will get you pumped up and ready for game day."/><meta property="og:type" content="product"/><meta property="twitter:card" content="summary_large_image"/><meta property="twitter:site" content="@Fanatics"/><meta property="og:site_name" content="Fanatics"/><meta property="og:url" content="https://www.fanatics.com/nba/los-angeles-lakers/lebron-james-los-angeles-lakers-jordan-brand-2021/22-number-6-swingman-player-jersey-purple-statement-edition/o-3536+t-58252968+p-60467601075+z-9-1901122059"/><meta property="og:price:amount" content="109.99"/><meta property="og:price:currency" content="USD"/><meta property="og:availability" content="In stock"/><meta name="theme-color" content="#FFFFFF"/><script>
```

Comment:
It looks little bit overkill, might be we can have smaller version of this web page?